### PR TITLE
Start using disable_ubsan in fuzz task.

### PR DESCRIPTION
**Pull request by inferno-chromium**
This is follow up to #1257

### CODE REVIEW

File: src/python/bot/tasks/fuzz_task.py [hunk 1]
```diff
@@ -665,6 +666,20 @@ def pick_redzone():
...
+  # This is only applicable in an ASan build.
+  memory_tool_name = environment.get_memory_tool_name(job_type)
+  if memory_tool_name not in ['ASAN', 'HWASAN']:
+    return False
+
+  # Check if UBSan is enabled in this ASan build. If not, can't disable it.
+  if not environment.get_value('UBSAN'):
+    return False
+
+  return not utils.random_number(0, DEFAULT_CHOOSE_PROBABILITY)
```
Comment (Dor1s) [inline src/python/bot/tasks/fuzz_task.py]: why only 10%? maybe 50?

File: src/python/tests/core/bot/tasks/fuzz_task_test.py [hunk 2]
```diff
@@ -730,6 +730,7 @@ def test_existing_unreproducible_testcase(self):
             job_type='job',
             fuzz_target=data_types.FuzzTarget(engine='engine', binary='binary'),
             redzone=111,
+            disable_ubsan=False,
```
Comment (Dor1s) [inline src/python/tests/core/bot/tasks/fuzz_task_test.py]: would it make sense to have a test to verify `get_ubsan_disabled_options` is working? All tests here and in #1257 are using `disable_ubsan=False`

File: src/python/bot/tasks/fuzz_task.py (hunk 1 above)
Comment (inferno-chromium) [reply src/python/bot/tasks/fuzz_task.py]: Starting slow. Also, i think for major targets we hurt fuzzing efficiency and lose ubsan bugs if we disable too many times.

File: src/python/tests/core/bot/tasks/fuzz_task_test.py (hunk 2 above)
Comment (inferno-chromium) [reply src/python/tests/core/bot/tasks/fuzz_task_test.py]: added.

**Pull request merged**